api/test/app.e2e-spec.ts

```diff
@@ -6,13 +6,14 @@ import { AppModule } from './../src/app.module';
 describe('AppController (e2e)', () => {
   let app: INestApplication;
 
-  beforeEach(async () => {
+  beforeEach(async (done) => {
```

Comment:
Maybe done is not required. I think done was required as workaround for some jasmine issue at one point, but I think its been fixed.  I noticed the error I had was actually failed to start due to not having database_url set first time I ran it.

Reply:
👍  I think they detect when you return a promise.

Reply:
I must push the fix as soon as possible :( for now we can add a require(..) as a temporary fix.